webservices/rest.py

```diff
@@ -155,31 +156,42 @@ def limit_remote_addr():
                 abort(403)
 
 
-def get_cache_duration(url):
+def get_cache_header(url):
 
+    # Time in seconds
     EFILING_CACHE = 0
     LEGAL_CACHE = 60 * 5
     CALENDAR_CACHE = 60 * 5
     DEFAULT_CACHE = 60 * 60
 
+    # cloud.gov time is in UTC (UTC = ET + 4 or 5 hours, depending on DST)
+    PEAK_HOURS_START = time(13)  # 9:00 ET + 4 = 13:00 UTC
+    PEAK_HOURS_END = time(23, 30)    # 19:30 ET + 4 = 23:30 UTC
```

Comment:
Would this calculation be manually updated based on EST and DST? Wondering if we could have a check to change that dynamically?

Reply:
@patphongs not without extra complexity and time. Because we're doing this for every API request, I think the benefit of a calculated value for DST doesn't outweigh the cost. We don't adjust any of our other processes (nightly updates, for example) for DST.

Reply:
@patphongs All our celery worker jobs run according to UTC time.  When after Daylight savings time adjustment all our schedule jobs start time also shift by 1 hour.